<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExerciseRandomiser</class>
 <widget class="QMainWindow" name="ExerciseRandomiser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExerciseRandomiser</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="taqbleCurrentExercise">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushNext">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableExerciseList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>631</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
